src/main/java/org/opensearch/knn/indices/ModelDao.java

```diff
@@ -219,21 +218,14 @@ public void create(ActionListener<CreateIndexResponse> actionListener) throws IO
             if (isCreated()) {
                 return;
             }
-            runWithStashedThreadContext(() -> {
```

Comment:
why we need to remove this from runWithStashedThreadContext?

Reply:
we do context stashing at the transport action level, which is a caller for this method, so keeping this functionality here is redundant.